spec/features/users/user_signs_up_and_signs_in_spec.rb

```diff
@@ -48,7 +50,7 @@
         fill_in :user_email, with: agent.email
         fill_in :password, with: agent.password
         click_on "Se connecter"
-        expect(current_path).to eq(organisation_agent_path(agent.organisation_ids.first, agent.id))
+        expect(current_path).to eq(organisation_home_path(agent.organisations.first))
```

Comment:
I think we should aim at being as explicit as possible in the specs, so avoiding using higher order helper methods like `organisation_home_path`. I'll change to `organisation_setup_checklist_path` here.